tutorials/s4sdk-resilience/s4sdk-resilience.md

```diff
@@ -90,26 +91,36 @@ import com.sap.cloud.sdk.s4hana.datamodel.odata.services.BusinessPartnerService;
 import com.sap.cloud.sdk.s4hana.datamodel.odata.services.DefaultBusinessPartnerService;
 
 public class GetBusinessPartnersCommand {
-
+    private static final Logger logger = LoggerFactory.getLogger(GetBusinessPartnersCommand.class);
     private static final String CATEGORY_PERSON = "1";
     private final ErpHttpDestination destination;
 
     private final BusinessPartnerService businessPartnerService;
     private final ResilienceConfiguration myResilienceConfig;
 
-    public GetBusinessPartnersCommand(String destinationName) {
-        this(destinationName, new DefaultBusinessPartnerService());
+    public GetBusinessPartnersCommand(ErpHttpDestination destination) {
+        this(destination, new DefaultBusinessPartnerService());
     }
 
-    public GetBusinessPartnersCommand(String destinationName, BusinessPartnerService service) {
-        destination = DestinationAccessor.getDestination(destinationName).asHttp().decorate(DefaultErpHttpDestination::new);
+    public GetBusinessPartnersCommand(ErpHttpDestination destination, BusinessPartnerService service) {
+        this.destination = destination;
         businessPartnerService = service;
 
-        myResilienceConfig = ResilienceConfiguration.of(BusinessPartnerService.class);
+        myResilienceConfig = ResilienceConfiguration.of(BusinessPartnerService.class)
+                .isolationMode(ResilienceIsolationMode.TENANT_AND_USER_OPTIONAL)
+                .timeLimiterConfiguration(
+                        ResilienceConfiguration.TimeLimiterConfiguration.of()
+                                .timeoutDuration(Duration.ofMillis(10000)))
+                .bulkheadConfiguration(
+                        ResilienceConfiguration.BulkheadConfiguration.of()
+                                .maxConcurrentCalls(20));        
     }
 
     public List<BusinessPartner> execute() {
-        return ResilienceDecorator.executeSupplier(this::run, myResilienceConfig, e -> Collections.emptyList());
+        return ResilienceDecorator.executeSupplier(this::run, myResilienceConfig, e -> {
+            logger.warn(e.getMessage());
```

Comment:
Doesn't that swallow stacktrace etc? I would do something like `logger.warn("Fallback called because of exception.", e);`